website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing Guidelines
+caption: Guidelines for writing consistent UI copy.
+description: Guidelines for writing consistent UI copy.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
```

Comment:
General comment (I know we discussed this in sync today, but I wasn't quite following the context until reading through this); this seems like it's heavily referencing the upcoming `Hds::Time` component, but doesn't explicitly reference it's existence, which is a bit confusing. Will there need to be some updating of this content when the `Time` component is released?

Reply:
Sorry, added reference to the Time component. The time component doc also references this.